core/src/main/scala/org/apache/spark/sql/TiContext.scala

```diff
@@ -70,8 +70,8 @@ class TiContext(val sparkSession: SparkSession, options: Option[TiDBOptions] = N
 
   final val version: String = TiSparkVersion.version
 
-  val autoLoad: Boolean =
-    conf.getBoolean(TiConfigConst.ENABLE_AUTO_LOAD_STATISTICS, defaultValue = true)
+  val autoLoad: Boolean = false
+//    conf.getBoolean(TiConfigConst.ENABLE_AUTO_LOAD_STATISTICS, defaultValue = true)
```

Comment:
delete this line

Reply:
will do so if all test cases passed. Right now, it is only a shortcut to speed debug.